docs/documenter_helpers.jl

```diff
@@ -0,0 +1,8 @@
+# this function is slightly more specific than the one from documenter, print the corresponding code location
+# calls the original function via invoke and prints a timing for the doctest
```

Comment:
I wonder if the Documenter people would be open to merging a similar change, possible hidden by a preference and off by default?